nvml/setup.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from codecs import open  # To use a consistent encoding
+from os import path
+
+from setuptools import setup
+
+HERE = path.dirname(path.abspath(__file__))
+
+# Get version info
+ABOUT = {}
+with open(path.join(HERE, 'datadog_checks', 'nvml', '__about__.py')) as f:
+    exec(f.read(), ABOUT)
+
+# Get the long description from the README file
+with open(path.join(HERE, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+def get_dependencies():
+    dep_file = path.join(HERE, 'requirements.in')
+    if not path.isfile(dep_file):
+        return []
+
+    with open(dep_file, encoding='utf-8') as f:
+        return f.readlines()
+
+
+def parse_pyproject_array(name):
+    import os
+    import re
+    from ast import literal_eval
+
+    pattern = r'^{} = (\[.*?\])$'.format(name)
+
+    with open(os.path.join(HERE, 'pyproject.toml'), 'r', encoding='utf-8') as f:
+        # Windows \r\n prevents match
+        contents = '\n'.join(line.rstrip() for line in f.readlines())
+
+    array = re.search(pattern, contents, flags=re.MULTILINE | re.DOTALL).group(1)
+    return literal_eval(array)
+
+
+CHECKS_BASE_REQ = parse_pyproject_array('dependencies')[0]
+
+
+setup(
+    name='datadog-nvml',
+    version=ABOUT['__version__'],
+    description='The nvml check',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    keywords='datadog agent nvml check',
+    # The project's main homepage.
+    url='https://github.com/DataDog/integrations-core',
+    # Author details
+    author='Jack Lindamood',
+    author_email='cep221@gmail.com',
```

Comment:
We will also have to update that part too. However, I think we should keep track of the fact that he is the original author of this integration but I'm not sure how to handle that. Maybe we could ask the team or #community?

Reply:
Either way, happy to pass ownership/"authorship" to someone else and great to see this moved to core.